docs/docs/api/config.md

```diff
@@ -0,0 +1,58 @@
+For multiple modules with common elements we can define a configuration to apply to all modules, or all modules of a certain type. Maintaining a common config helps reduce the code repitition and allows you to change settings accross many widgets in one place. A config object can be passed as third, optional argument to [initializeWidgets](/api/methods.md#initializewidgets).
```

Comment:
`passed as third, optional argument to` --> `passed as a third, optional argument to`